### ISSUE TITLE
Improve Accessibility in the Block List

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/gutenberg-mobile#14]
Users should be able to focus each block using the keyboard (or native accessibility features).

We should probably set the `accessible` attribute to `true` on the root `BlockHolder` element.

More info here: https://facebook.github.io/react-native/docs/accessibility.html

**Comment by guarani** [comment]
The Android-specific keyboard issue mentioned above seems related to native Aztec code on Android. @hypest suggested commenting out these two sections of code to see if they're causing the issue. 

https://github.com/wordpress-mobile/gutenberg-mobile/blob/dd2a9a7652a065d82b2c46fbaa5b6bd6f95e39a1/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java#L228-L229
and 
https://github.com/wordpress-mobile/gutenberg-mobile/blob/a4419c30be6781757e625290d286112d6dd38b79/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java#L107-L108

I commented these out, ran `yarn android`, and this indeed improves things—I'm now able to move focus between rich text areas using only the keyboard.

<details>
<summary>Screen recording</summary>
<img width=""250" src="https://user-images.githubusercontent.com/1898325/84696465-8f83a000-af1a-11ea-9ac2-92d9060283ef.gif">
</details>

I think this will need an Android developer to look into how to make this improvement permanent. Also, I noticed that when using tabs to navigate, the first tab highlights a block and the second puts the cursor on it (focuses it). When the block is highlighted (first tab press), subsequently hitting Space Bar perhaps should (?) allow you to using the block toolbar buttons (e.g. undo/redo/delete), but it sends the focus back up to the heading block, which seems incorrect.

<details>
<summary>Screen recording</summary>
<img width=""250" src="https://user-images.githubusercontent.com/1898325/84697524-4e8c8b00-af1c-11ea-83e7-19315a84a9b7.gif">
</details>

**Comment by hypest** [comment]
> but it instead sends the focus back up to the heading block, which seems incorrect

I think that's the first Aztec isntance _claiming_ focus, exactly the issue those line I suggest you try to remove are trying to combat actually. So, I think for this quick experiment, yes, it's expected to see this sideeffect when commenting them out.

**Comment by hypest** [comment]
This ticket is not worked on at the moment so, I'll clear the assignee field to denote so.
